src_to_mesh.cpp

```diff
@@ -118,14 +129,16 @@ int main(int argc, char** argv) {
   // generate a text output as CSV
   // Each row contains: OpenMC bin #, centroid (xyz), volume, rel. src strength
   std::ofstream output("mesh_src_strengths.csv");
-  output << "bin, cx, cy, cz, volume, rel. src" << std::endl;
+  output << "bin, cx, cy, cz, volume, rel. src, strength, vol. strength" << std::endl;
   for (int i = 0; i < mesh->n_bins(); i++) {
     std::stringstream line {};
     line << i << ", ";
     auto centroid = umesh_ptr->centroid(i);
     line << centroid.x << ", " << centroid.y << ", " << centroid.z << ", ";
     line << mesh->volume(i) << ", ";
-    line << rel_strengths[i] << std::endl;
+    line << rel_strengths[i] << ", ";
+    line << (rel_strengths[i]*total_strength) << ", ";
+    line << (rel_strengths[i]*total_strength/(mesh->volume(i))) << std::endl;
```

Comment:
This field is mainly for convenience?

Reply:
Yes, same as above I would like to be able to plot these values directly without having to run a secondary script. Not as big a deal for GeN-Foam, but it does allow you to back out the total_strength so you don't need to enter that value in the next script. Maybe a good middle ground is having cell strength but not volumetric strength

Reply:
Yeah one thing that's tricky is the total strength. I think ideally we'd have one more line at the top of the output file (or a separate, summary csv file?) that includes information like the total strength, the number of sites sampled, and the number of sources sites outside the mesh. 🤔